docs/2.guide/2.directory-structure/2.nuxtrc.md

```diff
@@ -0,0 +1,53 @@
+---
+title: ".nuxtrc"
+description: "The .nuxtrc file allows you to define nuxt configurations in a flat syntax."
+head.title: ".nuxtrc"
+navigation.icon: i-ph-file  
+---
+
+The `.nuxtrc` file can be used to configure Nuxt with a flat syntax. It is based on [`unjs/rc9`](https://github.com/unjs/rc9).
+
+::tip
+For more advanced configurations, use [`nuxt.config`](/docs/guide/directory-structure/nuxt-config).
+::
+
+## Usage
+
+```bash [.nuxtrc]
+# Disable SSR
+ssr=false
+
+# Configuration for `@nuxt/devtools`
+devtools.enabled=true
+
+# Add Nuxt modules
+modules[]=@nuxt/image
+modules[]=nuxt-security
+```
+
+If present, the properties in the `nuxt.config` file will overwrite the properties in `.nuxtrc` file.
```

Comment:
isn't this the other way around?

Reply:
In the implementation of `c12`, `config.main` is placed before `config.rc`, which means that the settings in `nuxt.config` take precedence over those in `.nuxtrc`.  

https://github.com/unjs/c12/blob/463386fdeaa3f1de36ddc67e420aa38d875b3181/src/loader.ts#L163-L170

Reply:
(also confirming that behavior for defu when it comes to defaults overriding other defaults)
<img width="732" alt="image" src="https://github.com/user-attachments/assets/2bef8be0-6453-4f86-98de-522cfad3354b" />